languages/js/src/Query.ts

```diff
@@ -1,13 +1,15 @@
-import type { Query as FfiQuery } from './polar_wasm_api';
+// TODO(gj): Make sure this is cool w/ Oso in the browser.
```

Comment:
```suggestion
```

I tested this by updating `package.json` in https://github.com/osohq/oso-demo-widget as follows:

```diff
diff --git a/package.json b/package.json
index f262ffe..b7bfdf7 100644
--- a/package.json
+++ b/package.json
@@ -18 +18 @@
-    "oso": "^0.20.0-beta",
+    "oso": "../oso/main/languages/js",
```

and then following the steps in the README: https://github.com/osohq/oso-demo-widget/blob/d58d3335ddff7d297e0160aa84fec5aae23675af/README.md#L9-L13

cc: @gkaemmer